pyServer/manifests/linux/hdinsight

```diff
@@ -11,4 +11,12 @@ copy,/etc/hive2/conf/*
 copy,/var/lib/.jupyter/jupyter_notebook_config.py
 
 echo,### HDInsight Host Files ###
-copy,/etc/hosts
\ No newline at end of file
+copy,/etc/hosts
+
+echo,### HDInsight Log Files ###
+copy,/var/log/ambari-server/ambari-server.log
+copy,/var/log/ambari-server/ambari-audit.log
+copy,/var/log/ambari-agent/ambari-agent.log
+copy,/var/log/hadoop-yarn/yarn/*
```

Comment:
Do you know? How big this can be? How many files?

Reply:
Each file collected there should be a few tens of MBs. I've pushed an update to just collect the .log files in the folder. Please take a look.